packages/react-examples/src/react-charting/AreaChart/AreaChart.AxisTitle.Example.tsx

```diff
@@ -0,0 +1,221 @@
+import * as React from 'react';
+import { AreaChart, ICustomizedCalloutData } from '@fluentui/react-charting';
+import { IAreaChartProps, ChartHoverCard } from '@fluentui/react-charting';
+import { ChoiceGroup, IChoiceGroupOption } from '@fluentui/react/lib/ChoiceGroup';
+import { Toggle } from '@fluentui/react/lib/Toggle';
+
+interface IAreaChartBasicState {
+  width: number;
+  height: number;
+  isCalloutselected: boolean;
+  showAxisTitles: boolean;
+}
+
+const options: IChoiceGroupOption[] = [
+  { key: 'basicExample', text: 'Basic Example' },
+  { key: 'calloutExample', text: 'Custom Callout Example' },
+];
+
+export class AreaChartAxisTitleExample extends React.Component<{}, IAreaChartBasicState> {
```

Comment:
Add the toggle in existing example only.

Reply:
done